src/components/Disputes/panels/AppealPanel.js

```diff
@@ -194,7 +194,7 @@ const AppealPanel = React.memo(function AppealPanel({
       )}
       <Info>
         Please note that if the final ruling outcome is different from your
-        selected appeal, the entire amount of your collateral will be slashed.{' '}
+        selected appeal, the entire amount of your collateral could be slashed.{' '}
```

Comment:
This sounds like we need a "learn more" link here.

Reply:
Added